src/resource-graph/azext_resourcegraph/vendored_sdks/resourcegraph/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.6.0"
+VERSION = "2.0.0"
```

Comment:
We should release 1.0.0 version for PS/CLI as there was no major version before. Also, we are not doing any change which breaks backward compat.

Reply:
It is python sdk